test/cpp_load_test.cpp

```diff
@@ -136,7 +136,7 @@ int main(int argc, char* argv[])
     try {
         sm.get_mem_fn<override_class, int()>("get");
         BOOST_TEST(false);
-    } catch(boost::dll::fs::system_error &) {}
```

Comment:
TODO: this may break user code. I have to keep throwing boost::system::error_code